src/com/hms_networks/americas/sc/thingworx/TWConnectorMain.java

```diff
@@ -208,19 +220,22 @@ public static void main(String[] args) {
         "Starting " + TWConnectorConsts.CONNECTOR_NAME + " " + TWConnectorConsts.CONNECTOR_VERSION);
 
     // Calculate local time offset and configure queue
+    String timeZoneName = "";
     try {
-      TWTimeOffsetCalculator.calculateTimeOffsetMilliseconds(
-          connectorConfig.getFtpUser(), connectorConfig.getFtpPassword());
-    } catch (JSONException e) {
+      timeZoneOffsetMilliseconds = SCTimeUtils.getTimeZoneOffsetMilliseconds();
+      timeZoneName = SCTimeUtils.getTimeZoneName();
+    } catch (Exception e) {
       Logger.LOG_WARN(
-          "An error occured retrieving FTP user credentials from the configuration file.");
+          "An error occurred while retrieving the Flexy's time zone information. UTC time (no"
+              + " offset) will be used!");
       Logger.LOG_EXCEPTION(e);
     }
-    final long calculatedTimeOffsetMilliseconds =
-        TWTimeOffsetCalculator.getTimeOffsetMilliseconds();
-    HistoricalDataQueueManager.setLocalTimeOffset(calculatedTimeOffsetMilliseconds);
+    HistoricalDataQueueManager.setLocalTimeOffset(timeZoneOffsetMilliseconds);
```

Comment:
Place in Try block. As this code is today this should not matter because timeZoneOffesetMilliseconds is initialized to 0.  But if it is ever changed and try throws and exception, this line might have unintended consequences.

Reply:
no longer applicable -- change removed